imdown-mobile/events/events.tsx

```diff
@@ -20,6 +20,7 @@ const Events = (props) => {
   const [events, setEvents] = useState(props.route.params.events)
   const squadName = props.route.params.squadName
   const squadEmoji = props.route.params.squadEmoji
+  const email = props.route.params.email
```

Comment:
Is there a reason why event would need email? I don't see it used anywhere. Or are you sending it just in case?

Reply:
Well we are using email to identify users instead of auth_hash so you'll need the email for the create_event endpoint. Plus as we implement features like google calendar, random group matchings, flake fee, might be useful to have.

Reply:
I actually think we should be able to identify the user from the session cookie only, but i can look into that later. seems fine to pass in email for now